types/rebass__grid/index.d.ts

```diff
@@ -34,10 +34,11 @@ export interface CommonProps {
     px?: ResponsiveProp;
     py?: ResponsiveProp;
     theme?: any;
+    css?: Interpolation<any>;
 }
 
 export interface BoxProps
-    extends Omit<React.HTMLProps<HTMLDivElement>, "width" | "wrap" | "is"> {
+    extends Omit<React.ComponentPropsWithRef<'div'>, "color" | "is">, CommonProps {
```

Comment:
This line is giving me an error, and I don't understand it. (It's also hard to get a minimal reproducible case, but it didn't show in one project but then all of a sudden did start)

```
node_modules/@types/rebass__grid/index.d.ts:40:18 - error TS2320: Interface 'BoxProps' cannot simultaneously extend types 'Pick<Pick<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "color" | "hidden" | "dir" | "slot" | "style" | "title" | "children" | "className" | "id" | "lang" | ... 240 more ... | "css"> & { ...; }, "hidden" | ... 249 more ... | undefined>' and 'CommonProps'.
  Named property 'css' of types 'Pick<Pick<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "color" | "hidden" | "dir" | "slot" | "style" | "title" | "children" | "className" | "id" | "lang" | ... 240 more ... | "css"> & { ...; }, "hidden" | ... 249 more ... | undefined>' and 'CommonProps' are not identical.

40 export interface BoxProps
                    ~~~~~~~~
```

I'll continue trying to get a reproducible case, but that may take a little time, I've spent wayyy to long trying to track this down already.